packages/cactus-plugin-satp-hermes/src/main/typescript/core/errors/errors.ts

```diff
@@ -9,7 +9,21 @@ export class SATPInternalError extends Error {
     super(message);
     this.name = this.constructor.name;
     Object.setPrototypeOf(this, new.target.prototype); // make sure prototype chain is set to error
-    this.stack = trace || new Error().stack;
+    //this.stack = trace || new Error().stack;
+  }
+}
+export class BridgeInternalError extends Error {
```

Comment:
this should probably be in bridge-errors. please rename this file to satp-errors